packages/ui/src/components/molecules/CardProfile.tsx

```diff
@@ -139,20 +131,25 @@ export const CardProfile: FC<TCardProfile> = ({
     justify="space-between"
     data-testid="card-profile"
   >
-    <UserInfoContainer direction="column" justify="flex-start">
-      <UserInfoWrapper>
+    <UserInfoContainer
+      direction="column"
+      justify="flex-start"
+      gap={dimensions.spacing.base}
+    >
+      <Avatar src={userData.profilePicture} alt={userData.profilePictureAlt} />
+      <UserInfoWrapper justify="space-between" gap={dimensions.spacing.xxs}>
         <UsernameStyled as="h1">{userData.userName}</UsernameStyled>
         <TextStyled>{userData.userEmail}</TextStyled>
         <ButtonStyled outline onClick={logoutData.handleLogOut}>
           {logoutData.logoutMsg}{' '}
-          <img
+          <ImgStyled
             src={logoutData.img?.logoutIcon}
             alt={logoutData.img?.altLogout}

```

Comment:
This img can be an Icon, maybe? 
I think Power Setting New fits: https://fonts.google.com/icons?selected=Material+Symbols+Outlined:power_settings_new:FILL@0;wght@400;GRAD@0;opsz@24&icon.query=power

Reply:
Yes, I will change it, but the actual icon that you say I will pass it in the correspondant organism, so that in CardProfile it is possible not to have icon for logout